[test_transformer_sd35.py] | (Galaxy) integration tests | galaxy-integration-tests / Galaxy SD3.5 integration tests [wh_galaxy]

## CI Failure Report

<!-- AUTO-TRIAGE-SUMMARY-START -->
**Summary:** [CI] Galaxy SD3.5 integration test fails with GatedRepoError / OSError accessing stabilityai/stable-diffusion-3.5-large on HuggingFace
<!-- AUTO-TRIAGE-SUMMARY-END -->

**Workflow:** (Galaxy) integration tests
**Job:** galaxy-integration-tests / Galaxy SD3.5 integration tests [wh_galaxy]

### Regression timeline

- **Last passing commit:** N/A (not provided)
- **First failing commit:** `c04ae2758fc8` — 2026-06-29T04:29:24Z — [job](https://github.com/tenstorrent/tt-metal/actions/runs/28348693408/job/83978531256)
- **Most recent failing commit:** `f880ac26fe2c` — 2026-06-30T04:24:10Z — [job](https://github.com/tenstorrent/tt-metal/actions/runs/28420168297/job/84212264871)

### Failing job URLs (last 2 runs)

- https://github.com/tenstorrent/tt-metal/actions/runs/28420168297/job/84212264871
- https://github.com/tenstorrent/tt-metal/actions/runs/28348693408/job/83978531256

### Failing test path(s)

- `models/tt_dit/tests/models/sd35/test_transformer_sd35.py::test_sd35_transformer_block[wormhole_b0-device_params0-1-4096-333-4x4sp0tp1]`

Triggered via: `./tests/scripts/tg/run_tg_frequent_tests.sh --model sd35`

### Error excerpt (terminal failure from logs)

**Run 1 (2026-06-30):**
```
huggingface_hub.errors.GatedRepoError: 401 Client Error. (Request ID: Root=1-6a43514f-68832f77370bb2d415be9e9d;01b5022f-ef2f-44d4-8ec1-6ac7b3d36096)

Cannot access gated repo for url https://huggingface.co/stabilityai/stable-diffusion-3.5-large/resolve/main/transformer/config.json.
Access to model stabilityai/stable-diffusion-3.5-large is restricted. You must have access to it and be authenticated to access it. Please log in.

/opt/venv/lib/python3.10/site-packages/huggingface_hub/utils/_http.py:787: GatedRepoError

OSError: stabilityai/stable-diffusion-3.5-large is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'

FAILED models/tt_dit/tests/models/sd35/test_transformer_sd35.py::test_sd35_transformer_block[wormhole_b0-device_params0-1-4096-333-4x4sp0tp1] - OSError: stabilityai/stable-diffusion-3.5-large is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'
================= 1 failed, 6 deselected, 3 warnings in 4.08s ==================
LOG_METAL: run_tg_frequent_tests failed
```

**Run 2 (2026-06-29):**
```
huggingface_hub.errors.GatedRepoError: 401 Client Error. (Request ID: Root=1-6a41fc5d-1867efa63415527b4dc306b1;943fa5cd-0011-4060-8984-944334b96fa3)

Cannot access gated repo for url https://huggingface.co/stabilityai/stable-diffusion-3.5-large/resolve/main/transformer/config.json.
Access to model stabilityai/stable-diffusion-3.5-large is restricted. You must have access to it and be authenticated to access it. Please log in.

/opt/venv/lib/python3.10/site-packages/huggingface_hub/utils/_http.py:787: GatedRepoError

OSError: stabilityai/stable-diffusion-3.5-large is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'

FAILED models/tt_dit/tests/models/sd35/test_transformer_sd35.py::test_sd35_transformer_block[wormhole_b0-device_params0-1-4096-333-4x4sp0tp1] - OSError: stabilityai/stable-diffusion-3.5-large is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'
================= 1 failed, 6 deselected, 3 warnings in 4.05s ==================
LOG_METAL: run_tg_frequent_tests failed
```

### Reproduction steps

```bash
# From the repository root, with appropriate environment setup:
./tests/scripts/tg/run_tg_frequent_tests.sh --model sd35

# Or run the specific failing test directly:
pytest models/tt_dit/tests/models/sd35/test_transformer_sd35.py \
  -k 'test_sd35_transformer_block[wormhole_b0-device_params0-1-4096-333-4x4sp0tp1]'
```

Note: Reproduction requires a TG (Galaxy) system with wormhole_b0 architecture and a valid HuggingFace token with access to the gated `stabilityai/stable-diffusion-3.5-large` model.

### Notes

- The test attempts to download the `stabilityai/stable-diffusion-3.5-large` model from HuggingFace, which is a **gated repository** requiring authentication and explicit access approval.
- The failure is a `GatedRepoError: 401 Client Error` — the CI runner either has no `HUGGING_FACE_HUB_TOKEN` set, the token has expired, or the token does not have access to this gated model.
- This is an infrastructure/credentials issue, not a code regression.

**Determinism check:**
- Run 1 terminal error: `FAILED models/tt_dit/tests/models/sd35/test_transformer_sd35.py::test_sd35_transformer_block[wormhole_b0-device_params0-1-4096-333-4x4sp0tp1] - OSError: stabilityai/stable-diffusion-3.5-large is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'`
- Run 2 terminal error: `FAILED models/tt_dit/tests/models/sd35/test_transformer_sd35.py::test_sd35_transformer_block[wormhole_b0-device_params0-1-4096-333-4x4sp0tp1] - OSError: stabilityai/stable-diffusion-3.5-large is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'`
- Both runs fail with the identical test ID, identical error class (`GatedRepoError` → `OSError`), and identical error message.

<!-- AUTO-TRIAGE-METADATA-START -->
`Auto-triage-workflow: (Galaxy) integration tests`
`Auto-triage-job-name: galaxy-integration-tests / Galaxy SD3.5 integration tests [wh_galaxy]`
<!-- AUTO-TRIAGE-METADATA-END -->

<!-- AUTO-TRIAGE-INFO-START -->
## About this issue (automated CI auto-triage)
This issue was opened and assigned by an automated AI system. It posts through a bot token, so notifications may look like they come from a person, but they are automated.

The owner is selected from recent commit history and job-ownership data. It reflects who is *likely able to fix or route* this, not necessarily who caused it, so it can be imperfect.

**If this was assigned to the wrong person, you can:**
- Comment `REASSIGN OWNER: <why this is outside your area / which team or test-area it belongs to>` and the system will automatically re-route it to a better-suited owner.
- Unassign yourself (this stops notifications to you).
- Close the issue if it is not a real or relevant failure.

The assigned owner is reminded about every 3 days until the issue is resolved. Full guide: https://tenstorrent.atlassian.net/wiki/spaces/MI6/pages/2545221692/Auto-Triage+Info
<!-- AUTO-TRIAGE-INFO-END -->

--- a/.github/workflows/galaxy-integration-tests-impl.yaml
+++ b/.github/workflows/galaxy-integration-tests-impl.yaml
@@ -88,11 +88,13 @@ jobs:
         PYTHONPATH: /work
         LD_LIBRARY_PATH: /work/build/lib
         LOGURU_LEVEL: INFO
+        HF_HUB_CACHE: /mnt/MLPerf/huggingface/hub
+        HF_TOKEN: ${{ secrets.HUGGINGFACE_TOKEN }}
       volumes:
         - ${{ github.workspace }}/docker-job:/work # Subdir to workaround https://github.com/actions/runner/issues/691
         - /dev/hugepages-1G:/dev/hugepages-1G
         # TG integration scripts use /mnt/MLPerf (HF) on functional and perf Galaxy SKUs.
-        - ${{ contains(matrix.test-group.sku, 'perf') && '/mnt/MLPerf:/mnt/MLPerf:ro' || '/donotmount:/donotmount:ro' }}
+        - /mnt/MLPerf:/mnt/MLPerf:ro
       options: "--device /dev/tenstorrent"
     defaults:
       run:

